source/runtime_gui.cpp

```diff
@@ -3204,12 +3204,28 @@ void reshade::runtime::draw_variable_editor()
 					const int ui_max_val = variable.annotation_as_int("ui_max", 0, ui_type == "slider" ? 1 : std::numeric_limits<int>::max());
 					const int ui_stp_val = std::max(1, variable.annotation_as_int("ui_step"));
 
+#define DESCRIPTION_DEFAULT_STRING "%d"
+					const size_t length_of_description = sizeof(DESCRIPTION_DEFAULT_STRING) - 1;
+					const size_t max_description_length = 11; // space + 10 characters
+					const size_t max_chars_for_description = max_description_length + 1; // +1 for \0
+					char description[max_chars_for_description] = DESCRIPTION_DEFAULT_STRING;
```

Comment:
At this point might as well just make these `std::string`s and avoid the offset and copy logic.

Reply:
Thought about that too.
Though having a limit on the maximum characters seems reasonable to me. Let me know what you think about that.